### DISCUSSION TITLE
Firefox rego dialogue linux missing

### DISCUSSION THREAD

**Discussion by Season4episode10** [aclap-dev/video-downloadhelper#1479] [Help and Q&A] [answered]
### Your question

I have switched to Linux (Bookworm Puppy (BookwormPup64_10.0.8) extension and co-app work flawlessly.

I can't find where to enter my license?!

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/root/.local/share/vdhcoapp/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unneeded":true}
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0
{

    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/root/.local/share/vdhcoapp"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "default_action": "download_as",
    "use_sidebar": true,
    "open_count_store": 4,
    "successfull_dl": 64,
    "never_show_successfull_dl_message": true
}

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/root/.local/share/vdhcoapp/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unneeded":true}
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0
{
  
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/root/.local/share/vdhcoapp"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "default_action": "download_as",
    "use_sidebar": true,
    "open_count_store": 4,
    "successfull_dl": 64,
    "never_show_successfull_dl_message": true
}

**Comment by Season4episode10** [comment]
license: {"unneeded":true}

Does this mean VDH is free for Linux users?

**Reply by Mattmjs** [reply, marked as answer]
Yes it is, when windows 10 is out of support next year I'm heading to Linux too.

**Comment by Season4episode10** [comment]
This laptop ran win7 without a single issue till I found a recently built lightweight distro that wasn't laggy, my experience with Linux recently is lightweight does not mean snappy or fast, though I'd used puppies in the past I was unaware that one had been created with Synaptic Package Manager, could install flatpaks and other self contained offerings, to get what I wanted from a puppy in the past I had to build it with what I wanted ... this is able to run everything that an Arch based distro can, this is the fastest OS that I can install every program I want that I've ever used.
